code/modules/clothing/spacesuits/swat.dm

```diff
@@ -1,9 +1,16 @@
 /obj/item/clothing/suit/space/swat
```

Comment:
Why not move it off of a space suit subtype

Reply:
so just stick it into armor.dm?

Reply:
moved into armor.dm